worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/ArgumentsManager.java

```diff
@@ -191,7 +252,12 @@ else if (type.equals("cd")){
 
         final ResolvedSetting resolvedSetting;
         try {
-            resolvedSetting = settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
+            if (shouldForceCacheRefresh(name, scopes, priorities, settingsServiceLastUpdateTimeMillisOpt)) {
+                resolvedSetting = forceCacheRefresh(
+                    () -> settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities)));
+            } else {
+                resolvedSetting = settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
+            }
```

Comment:
You can eliminate the duplication here:
```suggestion
            final SettingsServiceApiCall settingsServiceApiCall = ()
                -> settingsApi.getResolvedSetting(name, String.join(",", scopes), String.join(",", priorities));
            resolvedSetting = shouldForceCacheRefresh(name, scopes, priorities, settingsServiceLastUpdateTimeMillisOpt)
                ? forceCacheRefresh(settingsServiceApiCall)
                : settingsServiceApiCall.call();
```

Reply:
This was reworked a bit in the latest commit, code is a bit different now: https://github.com/CAFDataProcessing/worker-workflow/pull/88/commits/aa6fefce483b07010dfbb8aa1f5c35c4d47c1383